lib/sql_parser.js

```diff
@@ -72,6 +76,13 @@ function getFrom(query) {
   return { db: path[1], collection: path[2], name: name[1] };
 }
 
+function getJion(query) {
+  const from = query.match(/\sjoin\s+([\w([\])"'`.,/=<>\s]+$)/i)[1].replace(cutOff, '');
+  const name = from.match(/\s+as\s+(\w+)/i);
+  const path = from.slice(0, name.index).match(/(\w+)\/(\w+)/i);
```

Comment:
`path`, `name` and `from` are very similar to `getFrom` related variables. Can you move them into separate function or constants to follow DRY